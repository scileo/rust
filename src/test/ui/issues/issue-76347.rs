pub enum Foo {
    A(i32),
}

impl Foo {
    pub const A: Self = Foo::A(0);
    //~^ ERROR duplicate definition with name `A` [E0592]
}

fn main() {}
