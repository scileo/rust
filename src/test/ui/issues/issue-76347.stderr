error[E0592]: duplicate definition with name `A`
  --> $DIR/issue-76347.rs:6:5
   |
LL |     A(i32),
   |     ------ definition of variant `A`
...
LL |     pub const A: Self = Foo::A(0);
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ other definition of `A`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0592`.
